### PR TITLE
Fio single node benchmark test

### PR DESCRIPTION
**Pull request by garvct**
-  Script to install fio from git repo.

- PBS script to run an fio throughput and IOPS benchmark on a single Linux client.

- Powershell script to run the same fio I/O benchmark (throughput and IOPS) on a windows client.

### CODE REVIEW

File: apps/fio/readme.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# FIO
+
+Fio is a general I/O benchmark code, it has many options to configure and customize the I/O pattern being tested.
+
+To build a cluster with PBS, see one of the examples, e.g. [simple_hpc_pbs](../../examples/simple_hpc_pbs/readme.md)
+
+First log-in to the headnode.
+```
```
Comment (xpillons) [inline apps/fio/readme.md]: You should replace the connect and manual run by something like azhpc-run -u build_fio.sh.
Look at the readme.md in ProLB.

File: apps/fio/build_fio.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+prepend-path LD_LIBRARY_PATH ${INSTALL_DIR}/lib;
+prepend-path MAN_PATH ${INSTALL_DIR}/man;
+EOF
+}
+
+yum install -y zlib-devel
+cd $SHARED_APP
+git clone git@github.com:axboe/fio.git
+
+cd $APP_NAME
```
Comment (xpillons) [inline apps/fio/build_fio.sh]: Is there any prefered compiler version to use or the default one is good ?

Comment (garvct) [reply apps/fio/build_fio.sh]: Default compiler is good.

File: apps/fio/readme.md (hunk 1 above)
Comment (garvct) [reply apps/fio/readme.md]: ok, will make this change.

**Pull request merged**